app/src/main/java/com/example/development/sakaiclientandroid/utils/BottomNavigationViewHelper.java

```diff
@@ -0,0 +1,38 @@
+package com.example.development.sakaiclientandroid.utils;
+
+import android.annotation.SuppressLint;
+import android.support.design.internal.BottomNavigationItemView;
+import android.support.design.internal.BottomNavigationMenuView;
+import android.support.design.widget.BottomNavigationView;
+import android.util.Log;
+
+import java.lang.reflect.Field;
+
+/**
+ * Created by atharva on 7/4/18
+ */
+
+public class BottomNavigationViewHelper {
+
+
+    @SuppressLint("RestrictedApi")
```

Comment:
@akench Do not suppress warnings, they indicate that our app will immediately crash if methods are called on incompatible build versions. Wrap the concerning code in build version checks or find something that works all the way down to the minimum API version we are supporting.